frontend/src/app/shared/dialog/key-result-dialog/key-result-dialog.component.html

```diff
@@ -56,7 +65,7 @@
         [attr.data-testId]="'submit'"
         type="submit"
         (click)="saveKeyResult()"
-        [disabled]="keyResultForm.invalid"
+        [disabled]="keyResultForm.invalid || validOwner()"
```

Comment:
Find better solution for disabled. Form can be sent when enter is pressed on owner dropdown, even though form is not valid. Also rename validOwner method, since it returns false when owner is valid.

Reply:
Resolve this in other ticket